modules/special-rules/pages/psychic-beacon.adoc

```diff
@@ -8,10 +8,12 @@ Genestealer Patriarchs emit a strong psychic signal, which guides the parent Hiv
 
 When you field an army that includes detachments that have the {psychic-beacon} ability:
 
-* You can deploy any Mycetic Swarms that you have in the first turn, rather than roll for their arrival as Reserves.
-+
-{blank}[TODO: Is this the right interpretation? Seems like it might be overpowered -- include a single small Genestealer Cult detachment, and all Mycetic Swarms in a Tyranid army get to deploy on turn 1, no roll for Reserves required...]
+* For each undestroyed detachment with the {psychic-beacon} ability, you may re-roll one failed Reserves roll per turn.
```

Comment:
These seems too open, if I correctly understand the migration from the original rule to Remastered: Am I right to think that we should state something to the effect that this re-roll only applies to Mycetic Swarm detachments in reserve, so as not leave it open to _any_ Tyranid detachment in reserve...?

Reply:
Yes, the intention is that it is a re-roll for any detachment.

Reply:
Okay, you've deliberately opened it up as part of the new form this ability takes. Gotcha.

I'll draft an {e40k-diff} footnote to highlight this difference too.

Reply:
I'll resolve this conversation — the action to implement this continues in new conversation with suggestion at https://github.com/GameBrains/er-core/pull/36/files#r1111104113.